reaktive/src/jsMain/kotlin/com/badoo/reaktive/scheduler/CreateIoScheduler.kt

```diff
@@ -0,0 +1,4 @@
+package com.badoo.reaktive.scheduler
+
+actual fun createIoScheduler(): Scheduler =
+    throw NotImplementedError("IO scheduler is not supported in JS target")
```

Comment:
Perhaps it's better to fallback to main scheduler?

Reply:
Also agree with this. Will make using the library easier.